### ISSUE TITLE
[Oculus Normal/Heroic] Ley-Guardian Eregos - Despawn

### ISSUE THREAD

**Issue by arestavo** [dalaranwow/dalaran-wow#5721]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behavior**: Tell us what happens in detail (Where, When, What, Why, Who)
Ley-Guardian Eregos despawns during encounter and never comes back. This has happened two out of the last 3 times that I was in the Occulus Heroic mode.
**Expected behavior**: Tell us what should happen instead
He shouldn't despawn, and if he does he should come back.
**Steps to reproduce the problem**:

1. Summon any drake during encounter, or dismount during encounter, or have a party member that appears bugged (parachute) during the encounter, have any party member (at any time during the run, not just during the Ley-Guardian encounter) die, release and run back.
2. Ley-Guardian Eregos despawns never to return even after waiting 10 minutes, resetting the instance, all zoning out and in, etc.
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Lothloryen** [comment]
#2801
